### DISCUSSION TITLE
Is there a signal or other form of detection for a dialog ending?

### DISCUSSION THREAD

**Discussion by WinsomeCap** [dialogic-godot/dialogic#1211] [Q&A]
I was just wondering if there was a way to detect when a dialog or timeline ends?
I am trying to make the game pause when a dialog begins, and resume when it ends, so a signal or other way to detect this would be very helpful

**Comment by Jowan-Spooner** [comment]
Hi, this is likely very much to late for you, but for anyone else wondering: in 1.x is there is the timeline_end signal for this: https://github.com/coppolaemilio/dialogic/blob/dialogic-1/addons/dialogic/Documentation/Content/FAQ/Signals.md
